src/components/CartPage/CartPage.tsx

```diff
@@ -7,14 +7,17 @@ import styles from './CartPage.module.scss';
 
 export default function CartPage({}: Props) {
   const { itemCount, totalPrice } = useCartSelector(state => state);
+  const { cart } = useCartSelector(state => state);
 
   return (
     <main className={styles.cartPage}>
       <Breadcrumbs />
       <h1 className={styles.title}>Cart</h1>
       <div className={styles.cardsContainer}>
         <div className={styles.cardContainer}>
```

Comment:
I dont know if you misheard or smth? I repeated a bunch of times that you delete the containers in your component and wrap it around the cardContainer in the page and you didnt do either, making the layout completely pointless